direct/io-blob/src/main/java/cz/o2/proxima/direct/blob/BlobBase.java

```diff
@@ -13,10 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package cz.o2.proxima.direct.s3;
+package cz.o2.proxima.direct.blob;
 
-/** Force creation of -sources and -javadoc jars */
-public class Placeholder {
+import java.io.Serializable;
 
-  protected Placeholder() {}
+public interface BlobBase extends Serializable {
```

Comment:
-> `AbstractBlob`?

Can you please add javadoc as this is a public interface?

Reply:
Marked as `@Internal` as this is not intended for being used by users. Javadoc added.